inspectit-oce-bootstrap/README.md

```diff
@@ -0,0 +1,7 @@
+# inspectIT OCE - Bootstrap
+
+This project contains a manager class which will basically start the core of the inspectIT OCE Java agent.
+It also provides the `IAgent` interface which have to be implemented by each inspectIT OCE Java agent implementation.

```

Comment:
I would not mention any interface or class here.. I would mention that this project contains classes that need to be pushed to the bootstrap class loader and that these are usually interfaces that `oce-core` would implement. Going into the details, you can say that this is needed, as `core` is loaded by the inspectIT class loader and we need a bridge between user classes (loaded in application class loader) and inspectIT impl..